chimney/.jvm/src/test/scala/io/scalaland/chimney/JavaEnumSpec.scala

```diff
@@ -0,0 +1,291 @@
+package io.scalaland.chimney
+
+import utest._
+import io.scalaland.chimney.examples.JavaEnums
+import io.scalaland.chimney.examples._
+
+object JavaEnumSpec extends TestSuite {
+
+  val tests = Tests {
+
+    "transformerF" - {
```

Comment:
@krzemin , there's a duplication of tests betwee trasformer and transformerF. Do you have any suggestions on how to better approach this?